guides/review.rst

```diff
@@ -6,18 +6,29 @@ Note: This review guide was written with the Python version in mind,
 but should apply to *all* versions. If you find a place where it's a bit
 too Python specific, please fix it or file an issue.
 
-Hypothesis has a process of reviewing every change, internal or external.
-This is a document outlining that process. It's partly descriptive, partly
-prescriptive, and entirely prone to change in response to circumstance
-and need. We're still figuring this thing out!
+This document outlines the process for reviewing changes to Hypothesis. It's
+partly descriptive, partly prescriptive, and entirely prone to change in
+response to circumstance and need. We're still figuring this thing out!
+
+-----------------
+What Needs Review
+-----------------
+
+The repository includes Hypothesis implementations for multiple languages,
+which have different review requirements:
```

Comment:
I wonder if it would be worth clarifying that the review requirements largely correlate to maturity of the project. Maybe something like the `hypothesis-python` review requirements are for anything that has had a 1.0 release? Anyway, doesn't have to be as part of this, but we should probably think about what the policy will look like, especially once the shared rust back end is up and running.